lib/ts/recipe/passwordless/components/features/linkClickedScreen/index.tsx

```diff
@@ -76,8 +79,11 @@ const LinkClickedScreen: React.FC<PropType> = (props) => {
             }
 
             if (response.status === "RESTART_FLOW_ERROR") {
-                return SuperTokens.getInstanceOrThrow().redirectToAuthWithoutRedirectToPath(undefined, props.history, {
-                    error: "restart_link",
+                return SuperTokens.getInstanceOrThrow().redirectToAuth({
+                    history: props.history,
+                    queryParams: {
+                        error: "restart_link",
+                    },
```

Comment:
need to explicitly set redirectBack to false -> internal use of this function should always explicitly set this value to prevent future bugs.